app/src/main/java/com/dessalines/thumbkey/keyboards/DEMessageEaseSymbols.kt

```diff
@@ -15,6 +29,132 @@ import com.dessalines.thumbkey.utils.KeyboardC
+                        action = KeyAction.Undo,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.RIGHT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.AutoMirrored.Outlined.Redo),
+                        action = KeyAction.Redo,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.BOTTOM to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.ContentPaste),
+                        action = KeyAction.Paste,
+                        color = ColorVariant.MUTED,
+                    ),
+            ),
+        backgroundColor = ColorVariant.SURFACE_VARIANT,
+    )
+
+val KB_DE_MESSAGEEASE_SYMBOLS_EMOJI_KEY_ITEM =
+    KeyItemC(
+        center =
+            KeyC(
+                display = KeyDisplay.IconDisplay(Icons.Outlined.PanToolAlt),
+                action =
+                    KeyAction.SendEvent(
+                        KeyEvent(
+                            KeyEvent.ACTION_DOWN,
+                            KeyEvent.KEYCODE_ESCAPE,
+                        ),
+                    ),
+                size = FontSizeVariant.LARGE,
+                color = ColorVariant.SECONDARY,
+            ),
+        swipeType = SwipeNWay.EIGHT_WAY,
+        swipes =
+            mapOf(
+                SwipeDirection.TOP to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.Settings),
+                        action = KeyAction.GotoSettings,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.BOTTOM to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.Keyboard),
+                        action = KeyAction.SwitchIME,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.BOTTOM_LEFT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.Mic),
+                        action = KeyAction.SwitchIMEVoice,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.LEFT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.Language),
+                        action = KeyAction.SwitchLanguage,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.RIGHT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.LinearScale),
+                        action = KeyAction.SwitchPosition,
+                        color = ColorVariant.MUTED,
+                    ),
+            ),
+        backgroundColor = ColorVariant.SURFACE_VARIANT,
+    )
+
+val KB_DE_MESSAGEEASE_SYMBOLS_RETURN_KEY_ITEM =
```

Comment:
Please don't do a custom return key, use the common one. Or any additions and you will lose all functionality.

Reply:
I've tried to make it as close to MessagEase as possible.